### PR TITLE
Uploading files for the branch query-and-update-animals-table

### PR DESCRIPTION
**Pull request by siddghosh108**
- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

**TRANSACTIONS**
image

image

image

image

image

image

image

image

image

**QUESTIONS**

image

image

image

image

image

image

Please give your review
Thanks

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
```
Comment (Johnadibe) [inline queries.sql:21]: > - [ ] Good job @siddghosh108, According to the project requirements as stated [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20changes%20persist%20after%20commit.), after committing the transaction, you should verify that changes persist after committing. Please kindly do that by adding `SELECT * FROM animals` after line 21👍

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @siddghosh108👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ You used GitFlow correctly 👍🏽
✅ Good project documentation✔️
✅ Added a column species of type string to the animals table✔️
✅ Inserted the following data in data.sql file✔️
✅ In queries.sql, All transactions were implemented✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (siddghosh108) [reply queries.sql:21]: Made the changes as you suggested
BEGIN;
UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
SELECT * FROM animals;
COMMIT;
SELECT * FROM animals;

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @siddghosh108👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented the changes requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**